### PR TITLE
feature : add errorBorder to _MaterialTextFieldTheme

### PR DESCRIPTION
**Pull request by nick-beznos**
Corresponding issue -  #10

### CODE REVIEW

File: lib/theme/material_text_field_theme.dart [hunk 1]
```diff
@@ -136,6 +137,7 @@ class FilledOrOutlinedTextTheme extends _MaterialTextFieldTheme {
     Color? suffixIconColor,
     int? helperMaxLines,
     int? errorMaxLines,
+    InputBorder? errorBorder,
```
Comment (WaseemAbbas-DevCrew) [inline lib/theme/material_text_field_theme.dart]: here `InputBorder? errorBorder` should be replaced with `Color? errorBorderColor` as we did the same for `focusedBorder` and `enabledBorder`. This makes sure that all borders (i.e. errorBorder, focusedBorder, and enabledBorder) have the same border-radius and stoke.

File: lib/theme/material_text_field_theme.dart [hunk 2]
```diff
@@ -155,6 +157,7 @@ class FilledOrOutlinedTextTheme extends _MaterialTextFieldTheme {
           helperStyle: helperStyle,
           helperMaxLines: helperMaxLines,
           errorMaxLines: errorMaxLines,
+          errorBorder: errorBorder,
```
Comment (WaseemAbbas-DevCrew) [inline lib/theme/material_text_field_theme.dart]: Then here do like this:

```Dart
      errorBorder: OutlineInputBorder(
            borderSide: BorderSide(
                width: width, color: errorBorderColor ?? Colors.transparent),
            borderRadius: BorderRadius.circular(radius),
      ),
```

**Review by WaseemAbbas-DevCrew — CHANGES_REQUESTED** [review]

**Comment by nick-beznos** [comment]
Sorry for a such a long pause.

**Review by WaseemAbbas-DevCrew — APPROVED** [review]

**Comment by WaseemAbbas-DevCrew** [comment]
Nice work @nick-beznos. Thank you!

**Pull request merged**